### PR TITLE
Changing code structure

### PR DESCRIPTION
**Pull request by rafaelrubbioli**
- Changing the project structure to https://github.com/golang-standards/project-layout
- Removing vendor

### CODE REVIEW

File: makefile [hunk 1]
```diff
@@ -6,10 +6,10 @@ generate:
...
+	docker run -d -p 5000:5000 expenses:01
 
 rm:
-	docker delete expenses:01
+	docker stop $(docker ps -q --filter ancestor=expenses:01)
 
 run:
-	go run -mod=vendor main.go
\ No newline at end of file
+	go run cmd/server.go
```
Comment (rafaelsq) [inline makefile]: `cmd/server/server.go`

File: cmd/server.go [hunk 2]
```diff
@@ -5,20 +5,16 @@ import (
...
 func main() {
 	router := chi.NewRouter()
 	router.Use(dataloader.LoaderMiddleware)
 
-	router.Handle("/", handler.Playground("Expenses", "/query"))
-	router.Handle("/query", handler.GraphQL(
-		generated.NewExecutableSchema(generated.Config{Resolvers: &resolver.Resolver{}}),
-	))
+	router.Handle("/", graphql.HandlePlayground())
+	router.Handle("/query", graphql.Handle())
```
Comment (rafaelsq) [inline cmd/server.go]: `func` is only useful if you want to inject something on the handle.
Instead you can simple use the GraphQL Playground Handle for now.
```suggestion
	router.Handle("/query", handler.Playground("Expenses", "/query"))
```

**Comment by rafaelrubbioli** [comment]
@rafaelsq thanks for the tips!

**Review by tomtomesteves — APPROVED** [review]
LGTM

**Review by rafaelsq — APPROVED** [review]

**Pull request merged**